docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to get unleashed levels working in order to edit them, this guide does not cover how to use GLvl. its recommended you also read the [Generations GLvl guide.](/guides/hedgehog-engine/blueblur/levels/glvl.md){ target="_blank"}
```

Comment:
Punctuation seems a bit odd here, and link text should never include the sentence ending dot. Re-write this like so:

```
This guide covers how to setup Sonic Unleashed levels in GLvl, but does not cover how to use this tool. It's recommended that you also read the [Sonic Generations GLvl guide](/guides/hedgehog-engine/blueblur/levels/glvl.md).
```